nym-vpn-core/crates/nym-vpn-network-config/default/mainnet_discovery.json

```diff
@@ -1,5 +1,10 @@
 {
   "network_name": "mainnet",
   "nym_api_url": "https://validator.nymtech.net/api/",
-  "nym_vpn_api_url": "https://nymvpn.com/api/"
-}
+  "nym_vpn_api_url": "https://nymvpn.com/api/",
+  "feature_flags": {
+    "zkNyms": {
+      "credentialMode": "true"
```

Comment:
Could this not be problematic on eg canary etc where it's disabled?

Reply:
I've changed it to be the default of mainnet only